### PR TITLE
Password reset fixes

### LINKED ISSUE #7: Confirm password
Add a second box for the person to re-type the password; reject the submission if the two entries do not match.

### PR DESCRIPTION
**Pull request by kamfonik**
There are 2 updates in this PR:

* Require entering the email address twice, and check that the two fields match.
* Add simple CSS styling to the password and reset forms

~~Some validation-related modifications to password_form.html will come in a separate PR.~~ Actually, this was a simple enough change to add to this one.

Fixes #7

### CODE REVIEW

**Comment by kamfonik** [comment]
Oh right, tests. I should write a few of those for the reset form at some point.

File: setpass/api.py [hunk 1]
```diff
@@ -176,13 +180,17 @@ def view_reset_form():
...
+    confirm_email = request.form['confirm_email']
     pin = request.form['pin']
 
-    if not name or not username or not pin:
-        return Response(response='Missing name/pin/username!', status=400)
+    if not name or not email or not confirm_email or not pin:
+        return Response(response='Missing required field!', status=400)
+
+    if email != confirm_email:
+        return Response(response="Email addresses do not match.", status=400)
```
Comment (jeremyfreudberg) [inline setpass/api.py:191]: I'm not an expert or pedant regarding HTTP status code semantics, but the 'do not match' case seems to be inconsistently 400 or 403. (compare this line, 191, with 57)

File: setpass/templates/reset_form.html [hunk 2]
```diff
@@ -3,18 +3,38 @@
...
+        <div>
+            <label>Confirm email:</label>
+            <input name="confirm_email"/>
+        </div>
+        <div>
+            <label>PIN:</label>
+            <input name="pin"/>
+            <label class="note">You will provide this PIN later to complete the reset process.</label>
+        </div>
+   </fieldset>
```
Comment (jeremyfreudberg) [inline setpass/templates/reset_form.html]: It might be my eyes, but your tags in lines 35-37 don't seem to line up with the tag in line 17

**Review by jeremyfreudberg — APPROVED** [review]
Pre-approving: looks really good to me, just those minor comments

File: setpass/templates/reset_form.html (hunk 2 above)
Comment (lucashxu) [reply setpass/templates/reset_form.html]: ^

**Review by xuhang57 — APPROVED** [review]

**Comment by kamfonik** [comment]
I am going to hold off merging this until after the shutdown.

**Comment by jeremyfreudberg** [comment]
We should still merge/deploy this, yes?

**Comment by kamfonik** [comment]
@jeremyfreudberg yes, thanks for the reminder

**Pull request merged**